pkg/universe.dagger.io/go/test/configure.cue

```diff
@@ -0,0 +1,39 @@
+package go
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/go"
+	"universe.dagger.io/docker"
+	"universe.dagger.io/alpine"
+)
+
+dagger.#Plan & {
+	actions: test: {
+		_source: dagger.#Scratch & {}
+
+		simple: {
+			_goImg: docker.#Pull & {
+				source: "index.docker.io/golang:1.18-alpine"
+			}
+			_build: docker.#Build & {
+				steps: [
+					alpine.#Build & {},
+					go.#Configure & {
+						contents: _goImg.output.rootfs
+						source:   "/usr/local/go"
+					},
+				]
+			}
+
+			verify: docker.#Run & {
```

Comment:
should we validate that `GOROOT` and `GOPATH` are correctly set also?

Reply:
Added, also added a test for custom `GOROOT` and `GOPATH`